shared/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-authors = ["Jake McGinty <me@jake.su>"]
+authors = ["Jake McGinty <me@jakebot.org>"]
```

Comment:
```suggestion
authors = [
  "Jake McGinty <me@jakebot.org>",
  "Brian Schwind <brian@tonari.no>",
  "Ryo Kawaguchi <ryo@tonari.no>",
  "Matěj Laitl <matej@tonari.no>",
]
```

Reply:
Yep, I think best to keep it manual until cargo supports this deduplication natively.